assets/lora.html

```diff
@@ -2,91 +2,97 @@
 <html lang="en">
 	<head>
 		<meta charset="UTF-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>LORA Example</title>
-
+		<meta
+			name="viewport"
+			content="width=device-width, initial-scale=1.0, maximum-scale=1, viewport-fit=cover"
+		/>
+		<title>LORA Sample App</title>
 		<style>
-			/* Default for mobile screen */
-			#embed-lora-show {
-				width: 300px;
-				margin-top: 16px;
-				background-color: black;
-			}
-			/* For sreen size from 768px and up */
-			@media screen and (min-width: 768px) {
-				#embed-lora-show {
-					width: 400px;
-				}
-			}
-
-			/* For sreen size from 1024px and up */
-			@media screen and (min-width: 1024px) {
-				#embed-lora-show {
-					width: 500px;
-				}
-			}
-
-			.mt-4 {
-				margin-top: 16px;
+			body {
+				font-family: "Arial", sans-serif;
+				background-color: #f4f4f4;
+				margin: 0;
+				display: flex;
+				align-items: center;
+				justify-content: center;
+				min-height: 100vh;
 			}
 		</style>
 	</head>
+
 	<body>
-		<!-- <button id="btn-lora-show">Watch live</button>
+		<script src="https://lora-sdk.belive.sg/player-widget/staging/player.min.js"></script>
```

Comment:
Are we. releasing STG ?

Reply:
@hassanabidpk  We don't use that file any more, deleted it